Source/DafnyTestGeneration/README.md

```diff
@@ -77,47 +77,58 @@ public static Object[] mockParameters(Method method) {
 
 ## Example
 
-Suppose you have a file called `Char.dfy` with the following code:
+Suppose you have a file called `object.dfy` with the following code:
 ```dafny
-module Char {
-  method compareToB(c: char) returns (i: int) {
-    if (c == 'B') {
-        return 0;
-    } else if (c > 'B') {
-        return 1;
-    } else {
+module M {
+    class Value {
+        var v:int;
+    }
+    method compareToZero(v:Value) returns (i:int) {
+        if (v.v == 0) {
+            return 0;
+        } else if (v.v > 1) {
+            return 1;
+        }
```

Comment:
Use 2 spaces for indent. Same comment for L102-L116 below.

Reply:
Done!